src/triggers/messages.ts

```diff
@@ -0,0 +1,42 @@
+
+import { Table } from '../db/db';
+import { CrdtMessage, CrdtOpetation } from '../types/message';
+import { Cursor, Trigger } from '../types/trigger';
+
+export const pendingMempoolMessages: (tables: string) => Trigger<undefined> = 
+  (table) => {
+    return async (clients) => {
+      const sql = `
+      select rm.*, rcs."lastTimestamp"
+      from raw_mempool rm 
+      left outer join ${table} t 
+      on rm."entityId" = t.id
+      left outer join ${Table.crdtState} rcs
+      on rm."table" = rcs."table" 
+      and rm."column" = rcs."column" 
+      and rm."entityId" = rcs."entityId"
+      where rm."table" = '${table}'
+      and ((rm.operation = '${CrdtOpetation.UPDATE}' and t.id is not null) or rm.operation = '${CrdtOpetation.INSERT}')
```

Comment:
maybe we should just do inserts and updates in separate triggers/processors to simplify things?

Reply:
yeah I was also considering that approach. It could be nicer so I'll give it a go